newdle/api.py

```diff
@@ -200,15 +203,19 @@ def get_my_newdles():
     return MyNewdleSchema(many=True).jsonify(newdle)
 
 
-@api.route('/newdles/in')
-def get_newdles_im_in():
+@api.route('/newdles/participating')
+def get_newdles_participating():
     newdle = (
-        Newdle.query.options(noload('participants'))
-        .filter(Newdle.participants.any(auth_uid=g.user['uid']))
-        .order_by(Newdle.final_dt.isnot(None), Newdle.final_dt.desc(), Newdle.id.desc())
-        .all()
+        Participant.query.filter_by(auth_uid=g.user['uid'])
+        .join(Participant.newdle)
+        .order_by(
+            desc(Participant.answers == '{}'),
```

Comment:
Probably a matter of style, but you could just use `!=` instead of `==` here and then get rid of the `desc()` wrapper. Same in the next line by negating the check.

FWIW, I think both versions are somewhat ugly in terms of readability, but AFAIK there's no nicer way of sorting by the result of a comparison...

Reply:
Agree, I tried both approaches, though the previous was more explicit. But using a consistent format looks nicer.